cv-data-service-library/src/main/java/com/trihydro/library/model/WydotTim.java

```diff
@@ -9,16 +9,20 @@ public class WydotTim {
 

```

Comment:
question: might be outside the scope of this pr (backlog?) but do you think we should rename this class to remove the explicit connection to WYDOT?

Reply:
I think it would be a great idea to rename this class to remove the explicit WYDOT connection